library/general/src/modules/Arch.rb

```diff
@@ -528,6 +535,7 @@ def x11_setup_needed
     publish function: :is_zkvm, type: "boolean ()"
     publish function: :has_smp, type: "boolean ()"
     publish function: :x11_setup_needed, type: "boolean ()"
+    publish function: :is_wsl, type: "boolean ()"
```

Comment:
NP: Not sure if we are still publishing new functions, but is ok if not removed completely.

Reply:
Yes, you are right. I guess it is needed if you want to call the function from Perl or something like this. Anyway, I am keeping it just for consistency reasons.